packages/caravan-psbt/src/psbtv0/utils.ts

```diff
@@ -143,8 +143,9 @@ export const convertLegacyOutput = (output: LegacyOutput): PsbtOutput => {
     value: new BigNumber(output.amountSats).toNumber(),
     bip32Derivation:
       output.bip32Derivation || getBip32Derivation(output.multisig),
-    witnessScript: output.witnessScript || output.multisig?.witness?.output,
-    redeemScript: output.redeemScript || output.multisig?.redeem?.output,
+    witnessScript: output.witnessScript,
+    redeemScript: output.redeemScript,
```

Comment:
I think these are not necessary anymore right? they should come from `psbtMultisigLock`.

Reply:
If that's safe to do then yes! We can extract them from the `multisig` property—can update the fixtures such that this is handled smoothly